corpan/corpan-app/src/store/translations.ts

```diff
@@ -615,6 +623,83 @@ export const TRANSLATIONS = {
         "Instant polyglot practice": "即时多语言练习",
         "Show Romanization": "显示罗马化",
     },
+    "zh-Hant": {
```

Comment:
I'm not even sure what is required of this file, if anything. IIRC there is some dependency on it in maybe the onboarding but most of it is replaced with the proper `public/locales` i18n so.. this should either be pared wayy down or deleted altogether; here I just added all the stuff because I want to do other things right now.